src/ragapp/admin-ui/sections/config/providers/ollama.tsx

```diff
@@ -109,40 +117,53 @@ export const OllamaForm = ({
           Could not fetch Ollama models. Make sure the Ollama base URL is
           accessible with RAGapp.
         </FormMessage>
-      ) : getLLMModels(models ?? []).length === 0 ? (
+      ) : llmModels.length === 0 ? (
         <FormMessage>
           There is no LLM model available using Ollama. <br />
           Please pull a Ollama LLM model from &nbsp;
           <a href="https://ollama.com/library" target="_blank" rel="noreferrer">
             https://ollama.com/library
           </a>
         </FormMessage>
-      ) : getEmbeddingModels(models ?? []).length == 0 ? (
+      ) : embeddingModelsAvailable.length === 0 ? (
         <>
           <ModelForm
             form={form}
+            title="LLM Model"
+            description="Select an LLM model to use for language generation."
```

Comment:
why did you remove these lines?

Reply:
I think they are added for better description, not removed